Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.py

```diff
@@ -637,7 +723,7 @@ def update_assignment(self, assignee_id: str, assignment_id: str = None, account
             })
 
             # Execute request
-            return self._http_request(
+            return self.http_request(
                 method='POST',
                 url_suffix=API_ENDPOINT_ASSIGNMENT,
                 json_data=json_payload
```

Comment:
line 731 I would change to else and add a comment its host_id or add else to the method which raise an exception unsupported type. Currently the method does not return from all paths. if elif elif and no else or default.

Reply:
Hi @thefrieddan1, we have added else condition for this function if there no value supplied for any given parameters.

However, this endpoint function is used only in one of the command, where we are prechecking this condition.